ibm/service/cos/data_source_ibm_cos_backup_vault.go

```diff
@@ -0,0 +1,90 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cos
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	rc "github.com/IBM/ibm-cos-sdk-go-config/v2/resourceconfigurationv1"
+	"github.com/IBM/ibm-cos-sdk-go/aws"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceIBMCosBackupVault() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIBMCosBackupVaultRead,
+
+		Schema: map[string]*schema.Schema{
+			"backup_vault_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"service_instance_id": {
```

Comment:
Can we follow some inline naming convention between ibm_cos_* res/ds
We used resource_instance_id in cos_bucket